lib/common/http1client.c

```diff
@@ -681,7 +684,7 @@ static void on_connection_ready(struct st_h2o_http1client_t *client)
         return;
     }
 
-    h2o_iovec_t reqbufs[3];
+    h2o_iovec_t reqbufs[4];
```

Comment:
Can we simply set this to 5, as I think that would maximum regardless of how the branches are being taken? We could also adda comment here that the number of elements have been determined as such.

Reply:
Oh did I miss a case where 5 elements would be needed? Anyway I'll do that.

Reply:
Oh yeah I did miss one increment... actually 5 elements could be needed. Thanks for pointing out!

Reply:
Fixed in 51a63eba746dc7a30d7079edff3a4fad5e4d270c